cmd/oras/root/manifest/index/create.go

```diff
@@ -76,10 +76,10 @@ Example - Create and push an index with annotations:
   oras manifest index create localhost:5000/hello:v1 linux-amd64 --annotation "key=val"
 
 Example - Create an index and push to an OCI image layout folder 'layout-dir' and tag with 'v1':
-  oras manifest index create --oci-layout layout-dir:v1 linux-amd64 sha256:99e4703fbf30916f549cd6bfa9cdbab614b5392fbe64fdee971359a77073cdf9
-
+  oras manifest index create layout-dir:v1 linux-amd64 sha256:99e4703fbf30916f549cd6bfa9cdbab614b5392fbe64fdee971359a77073cdf9 --oci-layout
+  
```

Comment:
Need this space gone
```suggestion
 
```

I kind of think having the `--oci-layout` next to `layout-dir:v1` is more clear.

Reply:
The reason we changed this was because someone thought that `layout-dir:v1` was a argument to `--oci-layout`, but `--oci-layout` is actually a boolean flag and does not take any argument.

Reply:
Even though it isn't required, I think CLI examples in the form `cmd --flags arg1 arg2 ..` are clearer.